index.html

```diff
@@ -4,30 +4,153 @@
         <div class="hamburger"></div>
         <div class="hamburger"></div>
         <div class="hamburger"></div>
       </div>
     </nav>
   </header>
+  <!-- HEADLINE SECTION  -->
   <section class="headline">
     <h1>Nuri Lacka</h1>
     <h2>Software Developer</h2>
     <p>
       I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
     </p>
   </section>
+  <hr/>
+  <!-- LANGUAGES , FRAMEWORKS AND SKILLS SECTION  -->
+  <section class="languages">
+		<div class="languages-iner">
+			<ul>
+				<li><h3>Language</h3></li>
+				<li>Javascript</li>
+				<li>Ruby</li>
+				<li>Html</li>
+				<li>Css</li>
+			</ul>
+			<ul>
+				<li><h3>Frameworks</h3></li>
+				<li>Bootstrap</li>
+				<li>Ruby on Rails</li>
+				<li>RSpec</li>
+				<li>Capybara</li>
+				<li>Selenium</li>
+			</ul>
+			<ul>
+				<li><h3>Skills</h3></li>
+				<li>Codekit</li>
+				<li>Github</li>
+				<li>Codepen</li>
+				<li>Gitlab</li>
+				<li>Terminal</li>
+			</ul>
+		</div>
+	</section>
+  <hr/>
+  <!-- LANGUAGES , FRAMEWORKS AND SKILLS SECTION  -->
+  <section class="icons">
+    <div class="icons-iner">
+      <div class="social">
+        <a href="#" ><i class="fab fa-github"></i></a>
+        <a href="#" ><i class="fab fa-linkedin"></i></a>
+        <a href="#" ><i class="fab fa-angellist"></i></a>
+        <a href="#" ><i class="fab fa-twitter"></i></a>
+        <a href="#" ><i class="fab fa-medium-m"></i></a>
+      </div>
+      <div class="work-icon">
+        <i class="fas fa-arrow-down"></i>Work
+      </div>
+    </div>
+  </section>
+  <!-- WORK SECTION  -->
+  <section class="work">
+    <div class="work-iner">
+      <h1 class="work-title">MY RECENT WORK</h1>
+      <hr/>
+      <div class="card">
+        <img src="app_sceenshot.jpg" alt="mypage" />
+				<h3 class="card-header">Multi-Post stories</h3>
+				<p class="card-paragraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Doloribus, nam. Sint ut quisquam nihil voluptas.
+          Fugit omnis exercitationem dolorum praesentium sint nemo ea sequi, dolores facere molestias excepturi vero reiciendis.
+				</p>
+        <div class="techs">
+          <div class="tech1">css</div>
+          <div class="tech2">html</div>
+          <div class="tech3">bootsrap</div>
+          <div class="tech4">ruby</div>
+        </div>
+        <a href="@" class="btn btn-card">See Project </a>
+      </div>
+      <div class="card">
+        <img src="app_sceenshot.jpg" alt="mypage" />
+				<h3 class="card-header">Multi-Post stories</h3>
+				<p class="card-paragraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Doloribus, nam. Sint ut quisquam nihil voluptas.
+          Fugit omnis exercitationem dolorum praesentium sint nemo ea sequi, dolores facere molestias excepturi vero reiciendis.
+				</p>
```

Comment:
- [ ] One of the requirements says that all projects in my work section should be in the `<ul>` tag like this:
```
 <ul class="card-container">
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
</ul>
``` 
- [ ] the work container should also use the `grid` CSS property.

Reply:
Thank you, Anderson. Please check out my fixed commits and give me further instructions on what to do?